packages/react-native/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@novu/react-native",
+  "version": "2.2.0",
+  "repository": "https://github.com/novuhq/novu",
+  "description": "Novu's React Native SDK for building custom inbox notification experiences",
+  "author": "",
+  "license": "ISC",
+  "main": "dist/client/client.js",
+  "module": "dist/client/index.mjs",
+  "types": "dist/client/index.d.ts",
+  "browser": {
+    "./dist/server/server.js": "./dist/client/index.js",
+    "./dist/server/server.mjs": "./dist/client/index.mjs"
+  },
+  "files": [
+    "dist",
+    "dist/client/**/*"
+  ],
+  "sideEffects": false,
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build:watch": "tsup --watch",
+    "build": "tsup && pnpm run check-exports",
+    "lint": "eslint src",
+    "check-exports": "attw --pack .",
+    "release:preview": "pnpx pkg-pr-new publish"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@arethetypeswrong/cli": "^0.15.4",
+    "@types/node": "^20.14.12",
+    "@types/react": "*",
+    "@types/react-dom": "*",
+    "tsup": "^8.2.1",
+    "typescript": "5.6.2"
+  },
+  "peerDependencies": {
+    "react": ">=17",
+    "react-dom": ">=17"
+  },
+  "peerDependenciesMeta": {
+    "react-dom": {
+      "optional": true
+    }
+  },
+  "dependencies": {
+    "@novu/react": "workspace:*"
```

Comment:
@LetItRock will it do problems when published to npm? Do we replace it somehow later?

Reply:
@scopsy unfortunately it's a manual process where we have to replace this with the latest version prefixed with `^`

Reply:
@LetItRock is there something I need to do? Do we have some doc I need to update so that we will include this package aswell for the manual process?

Reply:
@scopsy nope, we don't have any docs, we just need to remember about this when we release